packages/components/src/Collapsible/Collapsible/Collapsible.module.scss

```diff
@@ -39,25 +35,17 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
 .container {
   background-color: $color-white;
   box-shadow: $shadow-small-box-shadow;
+  border: var(--border-width-1) var(--border-solid-border-style)
```

Comment:
We'll need to calc the border width from the padding

Reply:
As per Jess, to review all the border changes and adjust the padding so the height is the same as it was prior